.github/workflows/test_release.yaml

```diff
@@ -125,6 +125,13 @@ jobs:
       run: |
         cd examples
         ./test.sh
+    - name: Archive production artifacts
```

Comment:
will there be any logs? because I see they are cleaned up here: [test.sh#84](https://github.com/metacontroller/metacontroller/pull/578/files#diff-2107b9bb7be9ac26a632056369875d18f7f6091852d889612def0acdc7e54cf0R84)

Reply:
right, adjusted logic to keep the logs in CI